### PR TITLE
[FLINK-7135] [flip6] Pass in proper configuration to Dispatcher component

### PR DESCRIPTION
**Pull request by tillrohrmann**
This PR is based on #4259, #4260 and #4261.

The `Dispatcher` component should get passed in the proper `Configuration` instance from the `SessionClusterEntrypoint`.

### CODE REVIEW

File: flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobManagerRunner.java [hunk 1]
```diff
@@ -228,6 +228,7 @@ public JobManagerRunner(
 	//----------------------------------------------------------------------------------------------
 
 	public void start() throws Exception {
+		log.info("Start JobManagerRunner.");
```
Comment (zentol) [inline flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobManagerRunner.java]: intended change?

**Comment by zentol** [comment]
FYI, i integrated the dispatcher changes while merging #4260.

File: flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobManagerRunner.java (hunk 1 above)
Comment (tillrohrmann) [inline flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobManagerRunner.java]: Yes

**Comment by tillrohrmann** [comment]
@zentol thanks for the review. Also thanks for merging #4260. In the future I would refrain from taking partial changes and adding them to other PRs, because this can cause merge conflicts especially if the PR depends on other PRs.

**Pull request closed without merging**